src/preview/hover/utils.ts

```diff
@@ -15,6 +15,10 @@ export function mathjaxify(tex: string, envname: string, opt = { stripLabel: tru
     if (envname.match(/^(aligned|alignedat|array|Bmatrix|bmatrix|cases|CD|gathered|matrix|pmatrix|smallmatrix|split|subarray|Vmatrix|vmatrix)$/)) {
         s = '\\begin{equation}' + s + '\\end{equation}'
     }
+    if (envname == 'empheq') {
+        const empheqPat = /\\begin\{empheq\}\[.*?\]\{(\w+)\}/g;
+        s = s.replace(empheqPat, '\\begin{empheq}[]{$1}');
+    }
```

Comment:
We won't do special treatments to any particular macro. This need to be removed. A proper fix should be on MathJax side.

Reply:
I see.

The current implementation won't work properly for highly-ill formats like `\begin{empheq}[box=[\fbox]{test}]{align}`. You may need to use `getLongestBalancedString()` utility function in `src/utils/utils.ts` to extract the real m-arg.